test/libsolidity/util/SoltestTypes.h

```diff
@@ -229,6 +230,32 @@ struct FunctionCallArgs
 	}
 };
 
+/// Units that can be used to express function value
+enum class FunctionValueUnit
+{
+	Wei,
+	Ether
+};
+
+/// FunctionValue structure
+/// Holds value along with coin it is expressed in
+struct FunctionValue
+{
+	u256 value;
+	FunctionValueUnit unit = FunctionValueUnit::Wei;
+};
+
+inline bool operator==(FunctionValue const& _a, FunctionValue const& _b)
```

Comment:
Are these two operators needed?

Reply:
All the function call test rely on values to be equal.
I can either change all tests to look for _value.value and delete operator. Also if check fails it needs to print what is actual point of failure (which vales were not the same) .

Reply:
Do you actually need the unit to be the same?